packages/cosmos/buildinfo.json

```diff
@@ -5,8 +5,8 @@
   ],
   "single_source": {
     "kind": "url",
-    "url": "https://downloads.dcos.io/cosmos/0.6.1-SNAPSHOT-354-master-44225ce4d8/cosmos-server-0.6.1-SNAPSHOT-354-master-44225ce4d8-one-jar.jar",
-    "sha1": "8b04f1f148c573e3a3c55fa8b6e3e279f5cfc03d"
+    "url": "https://downloads.dcos.io/cosmos/0.6.1-29/cosmos-server-0.6.1-29-one-jar.jar",
```

Comment:
We can update master to use `0.6.2-SNAPSHOT`. This works too.

Reply:
will keep it as is for now, and update to master snapshot on our next cosmos merge